.changeset/real-bobcats-run.md

```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-checkbox-group": patch
+"@justeattakeaway/pie-checkbox": patch
+"pie-docs": patch
+---
+
+checkbox and checkbox-group components from alfa to beta
```

Comment:
```suggestion
[Updated] - checkbox and checkbox-group components from alfa to beta
```

Reply:
Also "alpha" not "alfa" 😄

Reply:
I'll update suggestion to fix this once

Reply:
Oops, sorry, one day I'll start to compose sentences in proper english, but it is not guaranteed 😄